pkg/api/meta/interfaces.go

```diff
@@ -142,4 +142,6 @@ type RESTMapping struct {
 type RESTMapper interface {
 	VersionAndKindForResource(resource string) (defaultVersion, kind string, err error)
 	RESTMapping(kind string, versions ...string) (*RESTMapping, error)
+	AddResourceAlias(alias string, resources ...string)
```

Comment:
This should not be public to consumers of RESTMapper.  You don't need to add the method here.